test/features/auth/presenters/login_presenter_test.dart

```diff
@@ -1,28 +1,85 @@
+import 'package:flutter_demo/core/domain/model/user.dart';
+import 'package:flutter_demo/features/auth/domain/model/log_in_failure.dart';
+import 'package:flutter_demo/features/auth/domain/use_cases/log_in_use_case.dart';
 import 'package:flutter_demo/features/auth/login/login_initial_params.dart';
 import 'package:flutter_demo/features/auth/login/login_presentation_model.dart';
 import 'package:flutter_demo/features/auth/login/login_presenter.dart';
+import 'package:flutter_demo/localization/app_localizations_utils.dart';
 import 'package:flutter_test/flutter_test.dart';
+import 'package:mocktail/mocktail.dart';
 
+import '../../../test_utils/test_utils.dart';
 import '../mocks/auth_mock_definitions.dart';
+import '../mocks/auth_mocks.dart';
 
 void main() {
   late LoginPresentationModel model;
   late LoginPresenter presenter;
   late MockLoginNavigator navigator;
+  late LogInUseCase logInUseCase;
 
   test(
-    'sample test',
+    'when password and username arent provided then button is enabled',
```

Comment:
are we sure ? I think it should be the opposite then
```suggestion
    'when credentials are empty then login button is disabled',
```